packages/firestore/test/util/test_platform.ts

```diff
@@ -58,7 +58,7 @@ export class FakeWindow implements WindowLike {
       case 'storage':
         this.storageListeners.push(listener);
         break;
-      case 'unload':
+      case 'pagehide':
       case 'visibilitychange':
         // The spec tests currently do not rely on `unload`/`visibilitychange`
```

Comment:
[nit] Looks like this comment refers to the old ‘unload’ rather than ‘pagehide’

Reply:
Thanks for catching this. WIll fix soon.